charts/gitops-server/templates/deployment.yaml

```diff
@@ -38,9 +38,9 @@ spec:
             - "{{ .Values.logLevel }}"
             {{- if .Values.serverTLS.enable }}
             - "--tls-cert-file"
-            - "/etc/tls-volume/tls.crt"
+            - {{ .Values.TLSCertFile }}
```

Comment:
I think the changes here for TLS need to be reverted, this was being configured via the `serverTLS` block in values.yaml which stored the TLS values in a k8s secret that was then mounted into the pod (hence the hard-coded path here)